pkg/controller/machinehealthcheck/machinehealthcheck_controller.go

```diff
@@ -25,7 +25,7 @@ import (
 )
 
 const (
-	machineAnnotationKey = "cluster.k8s.io/machine"
+	machineAnnotationKey = "machine.openshift.io/machine"
```

Comment:
Time to use https://github.com/openshift/cluster-api/blob/release-4.0/pkg/controller/node/node.go#L35 instead.

Reply:
Agreed, but that belongs to a different PR. We need to go step by step. Let's switch the api group. Then do improvements